source/howto/data/datum-alias.rst

```diff
@@ -0,0 +1,78 @@
+.. meta::
+    :description: This how to guide shows you how to create aliases for your data files and use them in your executions.
+
+.. _howto-datum-alias:
+
+Data aliases
+################################################
+
+You can create a data alias in your project and use the alias as the input for your executions.
```

Comment:
I would add to the end here that "Data aliases are aliases to Valohai datum URLs, enabling you to update the datab being used without any changes to code."

Not so much for the information content, but to mention datums in the beginning rather that suddenly at the end of this paragraph.